django-stubs/contrib/auth/views.pyi

```diff
@@ -9,6 +9,15 @@ from django.views.generic.edit import FormView
 
 UserModel: Any
 
+class RedirectURLMixin:
+    next_page: str | None
+    redirect_field_name: str
+    success_url_allowed_hosts: set[str]
+    def get_success_url(self) -> str: ...
+    def get_redirect_url(self) -> str: ...
+    def get_success_url_allowed_hosts(self) -> set[str]: ...
+    def get_default_redirect_url(self) -> str: ...
+
 class SuccessURLAllowedHostsMixin:
```

Comment:
Some more changes look warranted...

`SuccessURLAllowedHostsMixin` no longer exists in Django 4.2, `LoginView` and `LogoutView` should inherit from `RedirectURLMixin` instead.